app/services/organizations/update_service.rb

```diff
@@ -120,6 +120,13 @@ def handle_legacy_vat_rate(vat_rate)
     end
 
     def handle_eu_tax_management(eu_tax_management)
+      org_outside_eu = !organization.eu?
+      trying_to_enable_eu_tax_management = params[:eu_tax_management] && !organization.eu_tax_management
+      if org_outside_eu && trying_to_enable_eu_tax_management
+        result.single_validation_failure!(error_code: 'org_must_be_in_eu', field: :eu_tax_management)
+          .raise_if_error!
```

Comment:
Rubocop will prevent me to have only one line here or I'd have to use If modifier. With a modifier, the line is too long.

Reply:
you can do 
```ruby
return result.single_validation_failure!(error_code: 'org_must_be_in_eu', field: :eu_tax_management)
```

if you raise here, front end will have a 500 error and that's not the behavior we're waiting for